include/deal.II/fe/mapping.h

```diff
@@ -962,16 +962,15 @@ class Mapping : public Subscriptor
    *   this object also represents with respect to which cell the transformation
    *   should be applied to.
    * @param[out] output An array (or part of an array) into which the transformed
-   *   objects should be placed.
+   *   objects should be placed. (Note that the array view is @p const, but the
+   *   tensors it points to are not.)
```

Comment:
I had thought about this. I can't make the argument a non-`const` reference since then I can't bind the temporary produced by `make_array_view` to it. In C++11, I would make a non-`const` rvalue reference, but we can't do this here either. The only alternative would be to make it a non-`const` non-reference (passed by value). This is how the old code did it. But I don't find that particularly intuitive either, so I opted for the current approach.

Suggestions?